contributors/design-proposals/encryption.md

```diff
@@ -0,0 +1,204 @@
+# Encryption
+
+## Abstract
+
+The scope of this proposal is to ensure that resources can be encrypted at the datastore layer. Encrypting data at REST and via third party vendors is a desired feature but outside the scope of this proposal. Encryption will be optional for any resource but we suspect it will be used for the Secret resource in most cases.
```

Comment:
I think you can say encryption over the network is out of scope. API users should be using TLS connections. The kubelet can do that and there is work in flight to improve it.